cmd/podman/manifest/annotate.go

```diff
@@ -0,0 +1,56 @@
+package manifest
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/containers/libpod/cmd/podman/registry"
+	"github.com/containers/libpod/pkg/domain/entities"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	manifestAnnotateOpts = entities.ManifestAnnotateOptions{}
+	annotateCmd          = &cobra.Command{
+		Use:     "annotate",
+		Short:   "Add or update information about an entry in a manifest list or image index",
+		Long:    "Adds or updates information about an entry in a manifest list or image index.",
+		RunE:    annotate,
+		Example: `podman manifest annotate --annotation left=right mylist:v1.11 image:v1.11-amd64`,
+		Args:    cobra.ExactArgs(2),
+	}
+)
+
+func init() {
+	registry.Commands = append(registry.Commands, registry.CliCommand{
+		Mode:    []entities.EngineMode{entities.ABIMode, entities.TunnelMode},
+		Command: annotateCmd,
+		Parent:  manifestCmd,
+	})
+	flags := annotateCmd.Flags()
+	flags.StringVar(&manifestAnnotateOpts.OS, "os", "", "override the `OS` of the specified image")
+	flags.StringVar(&manifestAnnotateOpts.Arch, "arch", "", "override the `Architecture` of the specified image")
+	flags.StringVar(&manifestAnnotateOpts.Variant, "variant", "", "override the `Variant` of the specified image")
```

Comment:
Please lower case Variant and Architecture in the above lines.  I think this is what @rhatdan was asking.  I spot checked a few other podman commands, we're not uppercasing there.